src/locale/lt.js

```diff
@@ -24,6 +24,16 @@ function translateSeconds(number, withoutSuffix, key, isFuture) {
         return isFuture ? 'kelių sekundžių' : 'kelias sekundes';
     }
 }
+function monthsCaseReplace(m, format) {
+    var months = {
+            'nominative': 'sausis_vasaris_kovas_balandis_gegužė_birželis_liepa_rugpjūtis_rugsėjis_spalis_lapkritis_gruodis'.split('_'),
+            'accusative': 'sausio_vasario_kovo_balandžio_gegužės_birželio_liepos_rugpjūčio_rugsėjo_spalio_lapkričio_gruodžio'.split('_')
+        },
+        nounCase = (/D[oD]?(\[[^\[\]]*\]|\s+)+MMMM?/).test(format) ?
```

Comment:
Matches `D|Do|DD`, something custom (`[abc]` OR whitespace), then `MMMM`. I think the main thing is `D` followed by `MMM`.